pkg/virt-handler/non-root.go

```diff
@@ -117,7 +118,10 @@ func getTapDevices(vmi *v1.VirtualMachineInstance) []string {
 	for _, net := range vmi.Spec.Networks {
 		_, ok := macvtap[net.Name]
 		if ok {
-			tapDevices = append(tapDevices, net.Multus.NetworkName)
+			tapDevices = append(
+				tapDevices,
+				namescheme.HashNetworkName(net.Name),
+			)
```

Comment:
I wanted to avoid doing that since the naming scheme is applied for all interfaces on the VMI, and we're only looking into the macvtap ones.

Never the less, it might make it more readable. 

I am thinking of following this suggestion, but also push a follow-up where you can pass a predicate into the namescheme functions (it would only compute the pod interface name for interfaces meeting the predicate).
Thoughts @EdDev ?

Reply:
You could do that or just create a new function that does a lookup given the pool/map.
If the common usage does not require a predicate, it will be a bit noisy to require one from the majority of callers vs having another lookup function. But I am not sure how it will look like, maybe it will be fine.

Reply:
Done. For now I just did what you suggested. Let's refactor this in a follow-up.

Thanks for the discussion.